pwiz_tools/Skyline/Controls/Graphs/GraphSpectrum.cs

```diff
@@ -1501,6 +1502,11 @@ public MzRange Range
 
         public LibraryRankedSpectrumInfo SpectrumInfo => GraphItem?.SpectrumInfo;
 
+        public bool HasChromatogramData
+        {
+            get { return _spectrum?.LoadChromatogramData() != null; }
```

Comment:
This "LoadChromatogramData" call really does read the chromatogram from the sqlite database (in "Chromatogram.ReadLibrary"). I wonder whether GraphSpectrum should be hanging onto the chromatogram data so that it does not have to keep getting reloaded.